src/components/shared/CallNodeContextMenu.js

```diff
@@ -545,6 +556,19 @@ class CallNodeContextMenuImpl extends React.PureComponent<Props> {
               content: 'Merge node only',
             })}
 
+        {isProbablyJIT
+          ? this.renderTransformMenuItem({
+              l10nId:
+                'CallNodeContextMenu--transform-merge-unaccounted-native-functions',
+              shortcut: 'J',
```

Comment:
> you need to handle this shortcut to `handleCallNodeTransformShortcut` in `actions/profile-view`

Oops, thanks for catching that.

> May I ask why `J`? I think I'd use "u" or "U" for "unaccounted"? I guess this may be for "JIT" but because it's not mentioned anywhere it could be difficult to remember.

Yes, J was for JIT. My other motivation was to pick a shortcut that is unlikely to conflict with any other shortcut we may want to add in the future. I want this feature to be as non-intrusive as possible because I expect it to be rarely used.
However, now that I think about it, we may want to use `j` and/or `J` in the future for some JavaScript-related transform. So maybe `U` is a better option. It's also less surprising. I'll switch to it.